spec/features/sp_signup_spec.rb

```diff
@@ -2,6 +2,7 @@
 
 describe 'SP initiated sign up' do
   it 'creates new account via SP' do
+    pending "waiting for SP in prod"
```

Comment:
Do we want to disable this? I think we still look at it in the lower envs.

Reply:
Thanks for catching that too. I had that in there while I was testing in `prod`.  Eventually we'll have the demo SP running in prod so no changes will be needed.